TARGETS

```diff
@@ -919,6 +919,11 @@ ROCKS_TESTS = [
         "db/range_del_aggregator_test.cc",
         "serial",
     ],
+    [
+        "range_del_aggregator_bench",
```

Comment:
Since this is a benchmark, should we exclude it from the list of tests that will run in sandcastle?

Reply:
I'm removing it in #4426. Please let me know if I shouldn't do it. Thanks.